include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -135,26 +135,20 @@ SECTIONS
 
 	_image_rom_start = ROM_ADDR;
 
+/* Located in generated directory. This file is populated by the
+ * zephyr_linker_sources() Cmake function.
+ */
+#include <snippets-rom-start.ld>
```

Comment:
@oyvindronningstad it is now essential to add some information here about the sections before TEXT: what do they, generally contain? We need to say they contain the vector table, at least, so people won't be wondering where is the default Cortex-M exception table

Reply:
I added something to the comment. Note that I added a commit changing the wording of every comment to make them mention the relevant argument to `zephyr_linker_sources()`